client/src/Header.tsx

```diff
@@ -65,49 +61,32 @@ export const Header = () => {
 
 type OpenDrawerButtonProps = Required<Pick<UseDisclosureProps, 'onOpen'>>
 
+const DrawerIcon = () => <Icon as={AiOutlineMenu} boxSize={5} />
+
 const OpenDrawerButton: React.FC<OpenDrawerButtonProps> = ({ onOpen }) => (
-  <IconButton
-    variant='ghost'
-    size={'md'}
-    icon={<Icon as={AiOutlineMenu} boxSize={5} />}
-    aria-label={'Open Menu'}
-    onClick={onOpen}
-  />
+  <>
+    <Show below='md'>
+      <IconButton variant='ghost' size={'md'} icon={<DrawerIcon />} aria-label={'Open Menu'} onClick={onOpen} />
+    </Show>
+    <Show above='md'>
+      <Button variant='ghost' leftIcon={<DrawerIcon />} onClick={onOpen}>
+        <Show above='md'>Rooms</Show>
+      </Button>
+    </Show>
```

Comment:
This can be factored out into a component, and used for both the Rooms menu and the user button.